CEUS/CEUS_download.py

```diff
@@ -0,0 +1,39 @@
+"""
```

Comment:
Add more appropriate header information, e.g.:
```
__author__ = "Rob Knight"
__copyright__ = "Copyright 2019, SLAC"
__license__ = "GPL"
__version__ = "1.0.1"
__maintainer__ = "Rob Knight"
__email__ = ""
__status__ = "Beta | Alpha"
```

Not necessarily all of these, but you get the point.